scripts/create_compat.zs

```diff
@@ -0,0 +1,720 @@
+#snip modnotloaded create

```

Comment:
Requesting an author info header here

Reply:
Added something, I don't really know what info to put